oss/golang-gin/Makefile

```diff
@@ -0,0 +1,30 @@
+GO ?= go
+
+# Install the required dependencies
+install.deps:
+	@echo "==> Install dep"
+	@$(GO) get -u github.com/golang/dep/cmd/dep
+	@echo "==> Install dependencies"
+	@dep ensure
+.PHONY: install.deps
+
+# Install all the dependencies
+deps:
+	@echo "==> Install dependencies"
+	@dep ensure
+.PHONY: deps
+
+# Build all files.
+build:
+	@$(GO) build -o server ./...
```

Comment:
should be fine to leave this target out, I think most people `go run main.go`

Reply:
I have removed the makefile and have added a bit more to the readme